nidirect_gp/nidirect_gp.module

```diff
@@ -146,6 +146,33 @@ function nidirect_gp_gp_practice_form_validate(&$form, FormStateInterface $form_
   }
 }
 
+/**
+ * Submit callback for GP Practice node form.
+ */
+function nidirect_gp_gp_practice_form_submit(&$form, FormStateInterface $form_state) {
+  foreach (['field_gp_prescriptions', 'field_gp_appointments'] as $field_id) {
+    // If we have URL values for specific fields, set a fixed text value.
+    $field = $form_state->getValue($field_id);
+
+    if (empty(reset($field['uri']))) {
+      // Skip items with blank URI values.
+      continue;
+    }
+
+    // If we have a URI value then we want to use fixed labels for the link
+    // text on these fields.
+    if ($field_id == 'field_gp_prescriptions') {
+      $field[0]['title'] = t('Online appointments');
```

Comment:
Looks like you have a Copy 'n' Pasta error, should be 'Online prescriptions'

Reply:
D'oh... clipboard fail again!